src/Icon.js

```diff
@@ -0,0 +1,20 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Svg from '@moxy/react-svg';
+
+export const Icon = forwardRef(({ className, ...rest }, ref) => {
+    const finalProps = {
+        ...rest,
+        className: classNames('react-icon', className),
+    };
+
+    return <Svg ref={ ref } { ...finalProps } />;
```

Comment:
I may be wrong, but I think you still have to pass the ref as `forwardedRef`, as it is the prop the `Svg` component receives.